multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakServiceClient.java

```diff
@@ -71,9 +73,9 @@
 
   @Inject
   public KeycloakServiceClient(
-      KeycloakSettings keycloakSettings, OIDCInfoProvider oidcInfoProvider, JwtParser jwtParser) {
-    this.keycloakSettings = keycloakSettings;
-    this.oidcInfo = oidcInfoProvider.get();
+      @Nullable @Named(REALM_SETTING) String realm, OIDCInfo oidcInfo, JwtParser jwtParser) {
```

Comment:
sorry, I'm not following why are you changing the method signature and what is the `REALM_SETTING`, we do not have this setup on Hosted Che

Reply:
keycloakSettings put in the map `realm` from this variable, so we could inject it directly. But ok, I will revert this change, that not necessary.

Reply:
well, we can live if it is needed, but I suppose it will be null in Hosted Che case

Reply:
I removed this change. I guess I shouldn't do over refactoring.